model/riscv_insts_zvkned.sail

```diff
@@ -0,0 +1,550 @@
+/*
+ * Vector Cryptography Extension - NIST Suite: Vecttor AES Block Cipher
+ * ----------------------------------------------------------------------
+ */
+
+/*
+ * Helper functions.
+ * ----------------------------------------------------------------------
+ */
+
+val	 zvk_check_elements : (int, int, int, int) -> bool
+function zvk_check_elements(VLEN, num_elem, LMUL, SEW) = {
+  ((unsigned(vl)%num_elem) != 0) | ((unsigned(vstart)%num_elem) != 0) | (LMUL*VLEN) < (num_elem*SEW)
+}
```

Comment:
SAIL newbie question: the vector crypto spec distinguishes illegal cases (`(LMUL*VLEN) < (num_elem*SEW)`) and reserved cases (`((unsigned(vl)%num_elem) != 0) | ((unsigned(vstart)%num_elem) != 0)`) (which can legally raise an illegal exception or not). Does this difference is required in the sail model ?